The default of "/usr/var/cache/omod" won't be writable by normal users

--- a/src/omod_support.ml
+++ b/src/omod_support.ml
@@ -680,7 +680,7 @@
   let get_cache cache = match cache with
   | Some l -> l | None ->
       match get_env cache_env with
-      | Some l -> l | None -> in_prefix_path ("var" / "cache" / "omod")
+      | Some l -> l | None -> (let tmp = Filename.get_temp_dir_name () in try tmp / ".omod-cache-" ^ Unix.getlogin () with _ -> tmp / ".omod-cache")
 
   type t = { cache : Omod.fpath; libdir : Omod.fpath }
 
